Frameworks/UIKit/UILabel.mm

```diff
@@ -293,6 +200,7 @@ - (void)_initUILabel {
     }
 
     _textBlock = textBlock;
+    [_textBlock setLineStackingStrategy:WXLineStackingStrategyBlockLineHeight];
```

Comment:
no, I mean set it in the .xaml file on the TextBlock, so we don't have to call across here.

---
In reply to: [105782303](https://github.com/Microsoft/WinObjC/pull/2178#discussion_r105782303) [](ancestors = 105782303,105781932)